packages/components/pie-cookie-banner/src/index.ts

```diff
@@ -10,7 +10,30 @@ const componentSelector = 'pie-cookie-banner';
 
 export class PieCookieBanner extends LitElement implements CookieBannerProps {
     render () {
-        return html`<h1 data-test-id="pie-cookie-banner">Hello world!</h1>`;
+        return html`
+        <aside data-test-id="pie-cookie-banner" class="c-cookieBanner">
+            <p class="c-cookieBanner-title">Cookies</p>
```

Comment:
I'd probably make this a heading tag semantically. As it's in its own shadow dom, `h1` could actually make sense here?

Reply:
Fair point, I was very on the fence about using a heading here!

Reply:
It looks like the H1 would indeed appear on the document outline so I am unsure about heading level. Happy to chat about this more.